pkg/resolver/pkg.go

```diff
@@ -3,12 +3,19 @@ package resolver
 import (
 	"net/url"
 	"time"
-)
 
-const (
-	MaxContentLength = 1024 * 1024 * 5 // 5MB
+	"github.com/Chatterino/api/pkg/utils"
 )
 
+func GetMaxContentLength() {
```

Comment:
It's all in the same package, it should probably not be export aka. starting with lowercase